bindings/python/cedlib.pxd

```diff
@@ -1,4 +1,4 @@
-cdef extern from "edlib.h":
+cdef extern from "edlib.h" nogil:
```

Comment:
The GIL is Python's way of handling thread-safety.  Basically what it does is disable multiple threads operating concurrently, which makes everything thread-safe at the expense of any concurrency.  By turning the GIL off in line 72, we're allowing your code to run in a multi-threaded environment.  But what this means is that any C code that we call has to be annotated with nogil as well so that Cython knows it's OK to run that with the GIL disabled.

Reply:
http://docs.cython.org/en/latest/src/userguide/external_C_code.html#acquiring-and-releasing-the-gil